dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/AbstractImageRegistry.java

```diff
@@ -756,8 +756,7 @@ private void updateFiles(Tool tool, Tag tag, final FileDAO fileDAO, SourceCodeRe
             List<Checksum> checksums = new ArrayList<>();
             for (SourceFile newFile : newFiles) {
                 if (Objects.equals(oldFile.getAbsolutePath(), newFile.getAbsolutePath())) {
-                    oldFile.setContent(newFile.getContent());
```

Comment:
This copy constructor copies to and from json to make a copy. 
There may be performance implications vs setting two fields, but the other question is whether it changes the id (or other unintended fields) as well?